### PR TITLE
QF-1829 Assessor MI changes

### PR DESCRIPTION
**Pull request by ChrisJWoodcock**

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Web/Startup.cs [hunk 1]
```diff
@@ -229,7 +200,7 @@ private IServiceProvider ConfigureIoc(IServiceCollection services)
             {
                 config.Scan(_ =>
                 {
-                    _.AssemblyContainingType(typeof(Startup));
+                    _.AssembliesFromApplicationBaseDirectory(c => c.FullName.StartsWith("SFA"));
```
Comment (Jordan-fujitsu) [inline src/SFA.DAS.AssessorService.Web/Startup.cs:203]: @ChrisJWoodcock is this registration any services which the namespace starts with SFA?

File: src/SFA.DAS.AssessorService.Web/Helpers/EpaOrgIdFinder.cs [hunk 2]
```diff
@@ -8,6 +8,6 @@ public static class EpaOrgIdFinder
 
     public static string GetFromClaim(IHttpContextAccessor accessor)
     {
-        return accessor.HttpContext.User.FindFirst(ClaimType)?.Value;
+        return accessor?.HttpContext?.User?.FindFirst(ClaimType)?.Value;
```
Comment (Jordan-fujitsu) [inline src/SFA.DAS.AssessorService.Web/Helpers/EpaOrgIdFinder.cs:11]: @ChrisJWoodcock is there a reason why this has been updated?

File: src/SFA.DAS.AssessorService.Web/appsettings.json [hunk 3]
```diff
@@ -17,5 +17,6 @@
   "ConnectionStrings": {
     "Redis": ""
   },
-  "EnvironmentName": "LOCAL"
+  "EnvironmentName": "LOCAL",
+  "StubAuth": true
```
[the remark this answers is not in the document]
Comment (ChrisJWoodcock) [reply src/SFA.DAS.AssessorService.Web/appsettings.json:21]: The StubAuth when enabled is a local workaround for the GOV.UK One Login, when enabled it prompt you to enter details - a SignInId and Email which are in the Contacts table and then you are that user when you logon

File: src/SFA.DAS.AssessorService.Web/Startup.cs (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/SFA.DAS.AssessorService.Web/Startup.cs:203]: Yes - this is the same as used in the Api and External API already

**Review by Jordan-fujitsu — CHANGES_REQUESTED** [review]
@ChrisJWoodcock having issues running locally, seems like a DI error can you double check?

**Comment by ChrisJWoodcock** [comment]
> @ChrisJWoodcock having issues running locally, seems like a DI error can you double check?

This is not a comment for a PR

**Review by Jordan-fujitsu — APPROVED** [review]

**Pull request merged**